### DISCUSSION TITLE
comma in account balences

### DISCUSSION THREAD

**Discussion by jester223** [DSJAS/DSJAS#37] [General Q&A] [answered]
would it be possible to add comma to the account balences? it just looks odd espically for like larger amounts like 325252 listed in the bank looks odd on the site instead of it saying 325,252.  this is just an obervation that might help improve it.

**Comment by SinisterSpatula** [comment]
Here's a good piece of code that could accomplish it, if ethan wants to use it in the project:
```
formatMoney = (money, locale, currency) => {
  locale = locale || "en-US";
  currency = currency || "USD";
  if(money === undefined || money === null || isNaN(money)) { money = 0 };
  return money.toLocaleString(locale, { style: 'currency', currency: currency });
}
```
and then to use it:
`formatMoney(13)` or `formatMoney(1500, 'en-GB', 'GBP')` this way you can support users with different currencies and get automatic comma and decimal places.  Might need to switch the database field for money to integer, or DataTypes.DECIMAL(13, 2), toLocaleString expects a number or integer, not a string.

**Comment by ejv2** [comment, marked as answer]
Added as of 1e7fce6. Thanks for the suggestion

@SinisterSpatula Thanks for the code snippet. I've just used the built-in PHP function, but it does basically the same thing. I've added a wrapper for it to the theme API so that theme authors can format any currency they have to deal with.
